### PR TITLE
[Tour of Beam] add work example

### PR DESCRIPTION
**Pull request by sirenbyte**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py
Comment (riteshghorse) [inline learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py:56]: what are you intending to do with this statement?

File: learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py
Comment (riteshghorse) [inline learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py:36]: okay to remove the comment import if we are not using it

File: learning/tour-of-beam/learning-content/io/big-query-io/read-table/java-example/Task.java
Comment (riteshghorse) [inline learning/tour-of-beam/learning-content/io/big-query-io/read-table/java-example/Task.java:38]: To keep things clean I would recommend not using `.*` instead specific packages. 
May be someone from from java side can tell @Abacn

**Review by riteshghorse — CHANGES_REQUESTED** [review]

File: learning/tour-of-beam/learning-content/io/big-query-io/read-table/java-example/Task.java
Comment (Abacn) [reply learning/tour-of-beam/learning-content/io/big-query-io/read-table/java-example/Task.java:38]: Yes, beam code style forbids `.*` import: https://github.com/apache/beam/blob/master/sdks/java/build-tools/src/main/resources/beam/checkstyle.xml#L82

File: learning/tour-of-beam/learning-content/io/big-query-io/read-table/java-example/Task.java
Comment (riteshghorse) [inline learning/tour-of-beam/learning-content/io/big-query-io/read-table/java-example/Task.java:1]: The files always start with license. Please move this after the license

**Review by riteshghorse — CHANGES_REQUESTED** [review]

**Comment by riteshghorse** [comment]
Minor nit. But mostly looks good. Good to merge once this and merge conflicts are resolved.
Thanks!

**Review by riteshghorse — APPROVED** [review]
LGTM, thank you for your contribution!

**Comment by riteshghorse** [comment]
retest this please

**Comment by riteshghorse** [comment]
(comment for bot to trigger)

File: learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py
Comment (riteshghorse) [inline learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py:55]: I think indentation got changed to 2 spaces here

**Review by riteshghorse — CHANGES_REQUESTED** [review]
GCP IO check started failing from this commit and I think the accidental indentation change is the cause

File: learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py
Comment (sirenbyte) [reply learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py:55]: Spaces are necessary for the python file to work. @olehborysevych knows about the GCP error

Comment (sirenbyte) [reply learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py:55]: Remove spaces are necessary for the python file to work. @olehborysevych   knows about the GCP error

Comment (riteshghorse) [reply learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py:55]: @olehborysevych is there any issue created for the error?

Comment (riteshghorse) [reply learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py:55]: @sirenbyte can you comment on the known GCP error? we can merge this once we know the cause of failure

Comment (riteshghorse) [reply learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py:55]: great thanks for the info @olehborysevych

**Review by riteshghorse — APPROVED** [review]

**Pull request merged**
